WooCommerce/src/main/kotlin/com/woocommerce/android/ui/upgrades/UpgradesViewModel.kt

```diff
@@ -140,6 +164,10 @@ class UpgradesViewModel @Inject constructor(
             val daysLeftInFreeTrial: String
         ) : HasPlan
 
+        data class Upgradeable(
+            override val name: String
+        ) : HasPlan
```

Comment:
⚙️ Can we change the name of this state? We don't allow to upgrade plan in this scenario so, from our point of view, the plan is "nonupgradeable". Maybe

```suggestion
        data class PlanEnded(
            override val name: String
        ) : HasPlan
```

? Would work? But please feel free to suggest a different name

Reply:
I think it makes much more sense to use the PlanEnded naming pattern. Thanks for the suggestion!